src/users/pages/Members.vue

```diff
@@ -14,20 +14,63 @@
           :title="$t(sorting === 'joinDate' ? 'GROUP.SORT_NAME' : 'GROUP.SORT_JOINDATE')"
           @click="toggleSorting"
         />
-        <RouterLink
-          v-if="isEditor"
```

Comment:
Is the removal of checking for `isEditor` intentional? I think it makes sense. Just double checking ;)

Reply:
Yes. Inviting people by groupPreview link should be available to anyone in the group. :)